runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutorCompletableFutureTimeout.java

```diff
@@ -294,6 +285,21 @@ private ProcessBuilder preparePythonProcess(TempEnvironment executionEnvironment
         return processBuilder;
     }
 
+    private ProcessBuilder preparePythonProcessForEval(String pythonPath, String evalPyCode) {
+        // Must make sure that the eval.py evalPyCode does not contain the " character in its contents
+        // otherwise an error will be thrown when running python -c "import json\nimport sys..."
+        // code from eval.py separated using \n character
+        // Also do not use comments for the same reason in eval.py
+        // Use 'string' for Python strings instead of "string"
+        ProcessBuilder processBuilder = new ProcessBuilder(Arrays.asList(
+                Paths.get(pythonPath, "python").toString(),
+                "-c",
+                evalPyCode)
+        );
+        processBuilder.environment().clear();
+        return processBuilder;
+    }
+
```

Comment:
This code is duplicated with the one in its ScheduledExecutor counterpart. Maybe we should extract it in a utility class.

Reply:
I know about the duplication, let's have that in another PR.